### PR TITLE
Update strings comments

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Review by mcarare — APPROVED** [review]
LGTM! Waiting for @Delphine 's feedback.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -388,9 +388,9 @@
     <string name="customize_toggle_jump_back_in">Jump back in</string>
     <!-- Title for the customize home screen section with recently saved bookmarks. -->
     <string name="customize_toggle_recently_saved_bookmarks">Recently saved</string>
-    <!-- Title for the customize home screen section with recently visited. -->
+    <!-- Title for the customize home screen section with recently visited. Recently visited is a section were users see a list of tabs that they have visited in a few days -->
```
Comment (Delphine) [inline app/src/main/res/values/strings.xml:391]: Nits: 
* were > where
*  in a few days > in the past few days

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1439,7 +1439,8 @@
     <string name="etp_redirect_trackers_title">Redirect Trackers</string>
     <!-- Description of redirect tracker cookies that can be blocked by Enhanced Tracking Protection -->
     <string name="etp_redirect_trackers_description">Clears cookies set by redirects to known tracking websites.</string>
-    <!-- Description of the SmartBlock Enhanced Tracking Protection feature -->
+    <!-- Description of the SmartBlock Enhanced Tracking Protection feature. The * symbol is intentionally hardcoded here,
+         as we use it used it on the UI to indicate which trackers have been partially unblocked.  -->
```
Comment (Delphine) [inline app/src/main/res/values/strings.xml:1443]: nit: repetition of "as we use it"

**Review by Delphine — CHANGES_REQUESTED** [review]
Thanks for the fixes! just a few nits.
Also, could you please add in comments that example.com should not be translated, for the string that mentions that?
thank you

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1795,7 +1796,7 @@
     <string name="voice_search_explainer">Speak now</string>
     <!--  The error message in edit login view when a duplicate username exists. -->
     <string name="saved_login_duplicate">A login with that username already exists</string>
-    <!-- This is the hint text that is shown inline on the hostname field of the create new login page. -->
+    <!-- This is the hint text that is shown inline on the hostname field of the create new login page. 'https://www.example.com' intentionally hardcoded here -->
```
Comment (Amejia481) [inline app/src/main/res/values/strings.xml:1799]: @Delphine thanks for the review, I added that `example.com` is intentionally hardcoded

**Review by Delphine — APPROVED** [review]

**Pull request merged**